lldb/source/Target/SwiftLanguageRuntimeDynamicTypeResolution.cpp

```diff
@@ -1593,6 +1601,8 @@ bool SwiftLanguageRuntime::IsTaggedPointer(lldb::addr_t addr,
   if (!m_process)
     return false;
   swift::CanType swift_can_type = GetCanonicalSwiftType(type);
+  if (!swift_can_type)
```

Comment:
What's the reason for this early exit?

Reply:
This avoids a crash when the type is `nullptr`, however, the crash doesn't actually happen any more. It did happen with an earlier intermediate version that had bugs in it though.